docker-storage-setup.sh

```diff
@@ -1200,6 +1175,32 @@ if [ -e /usr/lib/docker-storage-setup/docker-storage-setup ]; then
   source /usr/lib/docker-storage-setup/docker-storage-setup
 fi
 
+# Main Script
+OPTS=`getopt -o h -l reset -l help -- "$@"`
```

Comment:
This patch is supposed to be adding tests. All this code was part of separate patch. Looks like you mixed up something.